packages/lab/src/tabs-next/TabNext.css

```diff
@@ -0,0 +1,122 @@
+/* Class applied to root Tab element */
+.saltTabNext {
+  --tabs-tab-background: var(--salt-navigable-primary-background);
+  --tabs-tab-background-hover: rgba(0, 0, 0);
+  --tabs-tab-backgroundOpacity-hover: 0.08;
+  --tabs-tab-color: var(--salt-text-primary-foreground);
+  --tabs-tab-letterSpacing: 0;
+  --tabs-tab-cursor: pointer;
+  --tabs-tab-position: relative;
+}
+
+.salt-theme[data-mode="dark"] .saltTabNext {
```

Comment:
Should there a mode dependent CSS in components?

Reply:
Unfortunately, I don't have these characteristics defined elsewhere yet. I can also add as a todo and an issue?